<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="TestFileButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test File</string>
    </property>
   </widget>
   <widget class="QLabel" name="SVMFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ParametrosCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DOF_Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="SVMFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SVM:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenVideo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenVideo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowTrack">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>38</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowTrack</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="VideoFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>9</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No File</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Parametros">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>261</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>247</width>
       <height>257</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="StartFrame">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="StartFrameLabel">
        <property name="text">
         <string>Start Frame</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="EndFrameLabel">
        <property name="text">
         <string>End Frame</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="TrajectorLengthLabel">
        <property name="text">
         <string>Trajectory Length</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="NeighborhoodSize">
        <property name="text">
         <string>32</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="NeighborhoodSizeLabel">
        <property name="text">
         <string>Neighborhood Size</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="SpatialCells">
        <property name="text">
         <string>2</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="TemporaryCells">
        <property name="text">
         <string>3</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="TemporaryCellsLabel">
        <property name="text">
         <string>Temporary Cells</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="ScaleNumber">
        <property name="text">
         <string>8</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="ScaleNumberLabel">
        <property name="text">
         <string>Scale Number</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="InitialGap">
        <property name="text">
         <string>1</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="SamplingStride">
        <property name="text">
         <string>5</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="TrajectoryLength">
        <property name="text">
         <string>15</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="SpatialCellsLabel">
        <property name="text">
         <string>Spatial Cells</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="EndFrame">
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="SamplingStrideLabel">
        <property name="text">
         <string>Sampling Stride</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="InitialGapLabel">
        <property name="text">
         <string>Initial Gap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>65</y>
      <width>535</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="ProcesarButton">
        <property name="text">
         <string>Procesar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="CrearDiccButton">
        <property name="text">
         <string>Crear Dicc.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="DOFDirectorioButton">
        <property name="text">
         <string>DOF Directorio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="DOFDirDiccButton">
        <property name="text">
         <string>DOF Dir. Dicc</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="ClusterDiccButton">
        <property name="text">
         <string>Cluster Dicc</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="TrainSVMButton">
        <property name="text">
         <string>Train SVM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SVM_Parameters">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>261</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>246</width>
       <height>344</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="svm_type"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>svm_type=100(C_SVC)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="kernel_type"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>kernel_type=2(RBF)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="degree"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>degree</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="gamma"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>gamma</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="coef0"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>coef0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="C"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="nu"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>nu</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="p"/>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>p</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="class_weights"/>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>class_weights</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="term_crit_type"/>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>term_crit_type=3(Iter+EPS)</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLineEdit" name="term_crit_max_iter"/>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>term_crit_max_iter</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLineEdit" name="term_crit_epsilon"/>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>term_crit_epsilon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="SVM_Parameters_Check">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SVM_Parameters</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Homography">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="Homo_X"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Homo_X_Label">
        <property name="text">
         <string>X_Divisions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="Homo_Y"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Homo_Y_Label">
        <property name="text">
         <string>Y_Divisions</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="Homography_checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Homography</string>
    </property>
   </widget>
   <zorder>SVM_Parameters</zorder>
   <zorder>TestFileButton</zorder>
   <zorder>SVMFile</zorder>
   <zorder>SVMFileName</zorder>
   <zorder>OpenVideo</zorder>
   <zorder>ShowTrack</zorder>
   <zorder>VideoFile</zorder>
   <zorder>Parametros</zorder>
   <zorder>splitter</zorder>
   <zorder>SVM_Parameters_Check</zorder>
   <zorder>Homography</zorder>
   <zorder>Homography_checkBox</zorder>
   <zorder>ParametrosCheckBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
